Demo/Demo/ViewModels/VaultState.swift

```diff
@@ -21,6 +22,14 @@ struct CardVaultState: Equatable {
         }
     }
 
+    var paypalVaultTokenResponse: LoadingState<String> = .idle {
+        didSet {
```

Comment:
I think I prefer the separate variables of loading states for each API response.
This way, I can have a result view to toggle errorView and successView.
I prefer separated result views here instead of toggling generic one with different options from the 
container view, especially with just a few different result views here that are shared across different payment methods.